polkadot/runtime/parachains/src/disputes/slashing.rs

```diff
@@ -696,7 +696,8 @@ where
 			key_owner_proof,
 		};
 
-		match SubmitTransaction::<T, Call<T>>::submit_unsigned_transaction(call.into()) {
+		let xt = <T as CreateInherent<Call<T>>>::create_inherent(call.into());
+		match SubmitTransaction::<T, Call<T>>::submit_transaction(xt) {
```

Comment:
In the future, the inherents will be the only bare transactions available. It's just the `Signed` variant that will go away and any bare (unsigned) transactions that are not inherents will be constructed using the `General` transaction type. You're right that the naming is not right, this should probably be  `submit_extrinsic` then?

Reply:
no I think only `ProvideInherent` should provide an inherent. In the future here it will create and send a general transaction.

Anyway naming will resolve once phase 2 is done.